tiingo/src/config.ts

```diff
@@ -0,0 +1,17 @@
+import { Requester } from '@chainlink/external-adapter'
+import { Config } from '@chainlink/types'
+
+export const DEFAULT_ENDPOINT = 'eod'
+export const DEFAULT_BASE_URL = 'https://api.tiingo.com/tiingo/'
+
+export const makeConfig = (prefix?: string): Config => {
+  const config = Requester.getDefaultConfig(prefix)
```

Comment:
Is the API key required for usage?

If so pass
```suggestion
  const config = Requester.getDefaultConfig(prefix, true)
```